tests/conftest.py

```diff
@@ -1,33 +1,29 @@
-import asyncio
 import warnings
-
 import pytest
-
 from bitcart import BTC
 from bitcart.coin import Coin
 
-TEST_XPUB = "xprv9yFGRF1GzMMydx5AK9FEU3UhqcHAmKZNSy8EvfBtztGvNtSfB1jLnkAJa4cvc33DJEpY5JXMDAdhN3fyanJJsNGcXSNsojKxzeX7EPEe8rg"
-
+TEST_XPUB_MAINNET = "xprv9yFGRF1GzMMydx5AK9FEU3UhqcHAmKZNSy8EvfBtztGvNtSfB1jLnkAJa4cvc33DJEpY5JXMDAdhN3fyanJJsNGcXSNsojKxzeX7EPEe8rg"
+TEST_XPUB_TESTNET = "tprv8ZgxMBicQKsPepFfedsYPCgWioGqkbnRbMmprdTq3jFmRf7JQwe3Yo8DMBwttKFNLpp3xVx6Rfv7ChxZbkLXgnmb8hcq4uN2hVKLmCNcTpB"
 
-@pytest.yield_fixture(scope="session")
-def event_loop(request):
-    loop = asyncio.get_event_loop_policy().new_event_loop()
-    yield loop
-    loop.close()
 
-
-@pytest.fixture(autouse=True, scope="session")
```

Comment:
Is this intended? I used autouse for session to use one instance per whole test session. Does having a fresh one each time have some benefits?

Reply:
It may cause a problem if other tests modified instance's attributes and do not reset properly. Also, create a new instance is not much expensive.

Reply:
Ok, good point. I agree